client/src/app/pages/login/login.component.scss

```diff
@@ -32,38 +24,32 @@ margin: auto;
         font-family: 'Roboto';
         font-style: normal;
         font-weight: 400;
-        font-size: 14px;
-        letter-spacing: 0.15px;
+        font-size: small;
+        letter-spacing: 0.10em;
         margin: .50em;
-        border-radius: 5px;
+        border-radius: .5em;
         border: 1px solid rgba(124, 109, 109, 0.87);
 
     }
 
-    i {
-        margin-top: -2em;
-        align-self: flex-end;
-        margin-right: 20px;
-    }
-
     button {
         display: flex;
         flex-direction: column;
         justify-content: center;
         align-items: center;
         padding: 0px;
-        gap: 8px;
-        width: 125px;
-        height: 40px;
+        gap: 5em;
+        width: 10em;
+        height: 3em;
         background: #2CBBC3;
         border-radius: 100px;
         border-color: #2CBBC3;
         border-style: none;
         font-family: 'Roboto';
```

Comment:
Remember to remove all the "font-family"  attributes from this file, as they override a rule with the same value

Reply:
done